bootstrap/src/main/java/gohigher/logging/QueryCountInspector.java

```diff
@@ -0,0 +1,35 @@
+package gohigher.logging;
+
+import org.hibernate.resource.jdbc.spi.StatementInspector;
+import org.springframework.stereotype.Component;
+
+@Component
+public class QueryCountInspector implements StatementInspector {
+
+	private final ThreadLocal<Long> queryCount = new ThreadLocal<>();
+
```

Comment:
```suggestion

	private static final ThreadLocal<Long> QUERY_COUNTER = new ThreadLocal<>();
```
전역적으로 한 개의 ThreadLocal을 사용하는 느낌을 부여하기 위해 `static`을 붙이는 것은 어떨까요?

Reply:
ThreadLocal에 대해서 좀 더 학습하고 바꿔보겠습니다!ㅜ